rsts/index.rst

```diff
@@ -3,41 +3,42 @@ Flyte
 
 .. image:: images/flyte_lockup_gradient_on_light.png
```

Comment:
I think we can get rid of this logo and the above

```
Flyte
=====
```

Title and have the main page start with `Meet Flyte`

With the updated theming I'll push up after this the branding will be clearer:

![image](https://user-images.githubusercontent.com/2816689/112322242-1e320400-8c87-11eb-8e8a-8516ddcb432a.png)

Reply:
Yes! Let's plan this with the new layout.